rbac/settings.py

```
@@ -0,0 +1,262 @@
+    'MAX_LOGIN_ATTEMPTS': getattr(settings, 'MAX_LOGIN_ATTEMPTS', 5),
+    'LOGIN_LOCKOUT_TIME': getattr(settings, 'LOGIN_LOCKOUT_TIME', 1800),  # 30 minutes
+    
+    # تنظیمات توکن JWT
+    'ACCESS_TOKEN_LIFETIME': getattr(settings, 'ACCESS_TOKEN_LIFETIME', 900),  # 15 minutes
+    'REFRESH_TOKEN_LIFETIME': getattr(settings, 'REFRESH_TOKEN_LIFETIME', 604800),  # 7 days
+    'JWT_ALGORITHM': getattr(settings, 'JWT_ALGORITHM', 'HS256'),
+    
+    # تنظیمات نشست
+    'SESSION_TIMEOUT': getattr(settings, 'SESSION_TIMEOUT', 7200),  # 2 hours inactivity
+    'MAX_SESSIONS_PER_USER': getattr(settings, 'MAX_SESSIONS_PER_USER', 5),
+    # aliasهای سازگاری
```

Ruff (0.12.2)

Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)